src/plugin_api.nim

```diff
@@ -577,7 +577,7 @@ proc checkPlugin(plugin: Plugin, codec: bool): bool {.inline.} =
 proc getAllPlugins*(): seq[Plugin] =
   var preResult: seq[(int, Plugin)] = @[]
   for name, plugin in installedPlugins:
-    preResult.add((plugin.configInfo.getPriority(), plugin))
+    preResult.add (get[int](plugin.configInfo, "priority"), plugin)
```

Comment:
nit
```suggestion
    preResult.add(get[int](plugin.configInfo, "priority"), plugin)
```

Reply:
this can confuse others. we use `()` predomindantly in chalk. please keep parens in